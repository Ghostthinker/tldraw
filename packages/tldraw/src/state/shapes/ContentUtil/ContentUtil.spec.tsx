import { Content } from '..'

describe('Edubreak Content Shape', () => {
  it('Creates a shape', () => {
    expect(Content.create).toBeDefined()
    expect(Content.create({ id: 'content' })).toMatchSnapshot('content')
  })
})
