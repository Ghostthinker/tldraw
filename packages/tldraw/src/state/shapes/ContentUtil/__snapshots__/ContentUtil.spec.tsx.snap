// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Edubreak Content Shape Creates a shape: content 1`] = `
Object {
  "body": "body",
  "childIndex": 1,
  "id": "content",
  "name": "Content",
  "nodeType": "",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "rotation": 0,
  "size": Array [
    400,
    400,
  ],
  "style": Object {
    "color": "black",
    "dash": "draw",
    "font": "script",
    "isFilled": false,
    "scale": 1,
    "size": "small",
    "textAlign": "middle",
  },
  "title": "title",
  "type": "content",
}
`;
